javascript/chronometer.js

```diff
@@ -1,26 +1,43 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime= 0;
+    this.intervalId= 0;
+
   }
   startClick(callback) {
-    // ... your code goes here
-  }
-  getMinutes() {
-    // ... your code goes here
-  }
-  getSeconds() {
-    // ... your code goes here
-  }
-  twoDigitsNumber() {
-    // ... your code goes here
-  }
+
+this.intervalId = setInterval(() => this.currentTime++;
```

Comment:
If you need to run multiple lines inside a function you need to start with curly brackets. Missing "{" after the arrow function.